core/lib/web3_decl/src/namespaces/zks.rs

```diff
@@ -110,4 +110,12 @@ pub trait ZksNamespace {
 
     #[method(name = "getLogsWithVirtualBlocks")]
     async fn get_logs_with_virtual_blocks(&self, filter: Filter) -> RpcResult<Vec<Log>>;
+
+    #[method(name = "zks_getProof")]
```

Comment:
Shouldn't the name be simply `getProof`? Other method names don't have the `zks_` prefix.